README.md

```diff
@@ -14,3 +35,269 @@ at least tested with.
 Z points to bugfix updates to this role itself, and does not depend on any bugfix
 release from upstream or any upstream version change.
 
+
+Example Playbook
+----------------
+
+Using `group_vars/seafile.yml`:
+(beware, this is just an old example!)
+
+```
+---
+# names, files and directory locations
```

Comment:
We should remind users to set `seafile_install_version` explicitly
Also I just noticed I need to bump it in the example group_vars as well